tests/test_part_image.py

```diff
@@ -374,6 +378,7 @@ def test_expected_data_dirs(
                 os.close(old_cwd_fd)
 
 
+@pytest.mark.cross_platform
```

Comment:
Maybe not this one? It has quite a lot of image labels attached to it, and the generation depends on the wic generator, which I know has different code paths for different partition types.